### PR TITLE
feat(doc): Improve installation doc

### PR DESCRIPTION
**Pull request by yngvark**
Make setup instructions for okcl.io + README better for okctl beginners.

TL;DR: See changes in

* okctl.io new front page: click netlify link on bottom of this PR
* new readme preview

## Description

Improves the user experience of setting up a cluster for the first time, by improving the quality and simplicity of installation section in the okctl.io front page and README.

https://trello.com/c/nKXhx8Mw/98-improve-usability-from-usage-testing

## Motivation and Context

User testing on multiple people has shown that parts of the documentation is not good enough, people get stuck and has to resort to asking us for help.

The new readme and okctl.io front page will with the links to AWS login (keycloak Oslo Kommune) be a bit more specific to Oslo Kommune. As a consequence. the site is less general and perhaps less fork-able, but I think it is neccessary and worth it to provide a good user experience.

## How Has This Been Tested?

The Netlify CI build and readme has been visually inspected.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have updated the release notes (for the next release).

### CODE REVIEW

**Review by deifyed — APPROVED** [review]

**Review by ivaruf — APPROVED** [review]
Looks very good :tada:
